modules/items/client/views/list-items.client.view.html

```diff
@@ -1,62 +1,47 @@
 <section>
   <div class="page-header">
-    <div class="row">
-      <div class="col-xs-4">
-        <h1>
-          Items
-        </h1>
-      </div>
-      <div class="col-xs-4 pull-right">
-        <a id="add-item-btn" class="btn btn-primary pull-right" data-ui-sref="items.create">
-          <i class="glyphicon glyphicon-plus"></i>
-        </a>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-xs-12">
-        <input class="form-control"
-          type="text"
-          placeholder="Search by name, content, category or module" 
-          ng-model="query"/>
-      </div>
-    </div>
+    <h1>
+      Items
+      <a class="btn btn-primary pull-right" data-ui-sref="admin.items.create">
+        <i class="glyphicon glyphicon-plus"></i>
+      </a>
+    </h1>
   </div>
-  <div class="list-group">
-    <a data-ng-repeat="item in vm.items | filter: query" data-ui-sref="items.edit({itemId: item._id})" class="list-group-item item-box">
-      <h4 class="list-group-item-heading" data-ng-bind="item.title"></h4>
-      <p class="list-group-item-text item-content" data-ng-bind="item.content"></p>
-      <div ng-if="item.categories" class="list-group-item-text">
-        Category:&nbsp;
-        <span data-ng-bind="item.categories.title"></span>
-      </div>
-      <div ng-if="item.modules" class="list-group-item-text">
-        Module:&nbsp;
-        <span data-ng-bind="item.modules.title"></span>
-      </div>
-      <div>
-        Count:&nbsp;
-        <span ng-if="item.count" ng-bind="item.count"></span>
-      </div>
-      <div>
-        Status:&nbsp;
-        <span ng-if="item.workingStatus" data-ng-bind="item.workingStatus"></span>
-      </div>
-      <div>
-        Posted on
-        <span data-ng-bind="item.created | date:'mediumDate'"></span>
-        by
-        <b>
-          <span ng-if="item.user" ng-bind="item.user.displayName"></span>
-          <span ng-if="!item.user">Deleted User</span>
-        </b>
-      </div>
-    </a>
+  <div class="list-group" style="width: 100%">
+    <table>
```

Comment:
are you able to put the buttons in the bottom of the a tag instead of putting them in a table? this screws up a lot of the css. If you just put the buttons at the bottom, I can work with that better